libraries/rush-lib/src/schemas/version-manifest.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "Version manifest output file format",
+  "description": "A version manifest is an output file produced by the 'rush version' command in Rush, describing the set of projects that have new versions and should be published and/or deployed.",
+  "type": "object",
+  "required": ["projects"],
+  "additionalProperties": false,
+  "properties": {
+    "$schema": {
+      "description": "Part of the JSON Schema standard, this optional keyword declares the URL of the schema that the file conforms to. Editors may download the schema and use it to perform syntax highlighting.",
+      "type": "string"
+    },
+    "projects": {
+      "description": "A list of versioned projects.",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": ["name", "relativeFolderPath", "version"],
+        "additionalProperties": false,
+        "properties": {
+          "name": {
+            "description": "The name of the versioned project.",
+            "type": "string"
+          },
+          "relativeFolderPath": {
+            "description": "The path to the folder containing this project, relative to the root of the monorepo.",
+            "type": "string"
+          },
+          "version": {
```

Comment:
I am thinking about whether "version" and "previousVersion" is useful here.

Assuming the we have a `my-lib@1.0.0`, and going to do a patch version.

Running `rush version` generate the publishing plan file, including

```
[
  {
    "name": "my-lib",
    "version": "1.0.1",
    "changes": [...],   // <-- I can see "comments" property in design, where is it? 🤔
  }
]
```

But, in the workflow, the version is not the "final" version during publishing. By running `rush publish --plan <plan_file_above> --suffix alpha.0`. (Correct me if I am wrong).

The `version`s here in the manifest becomes less useful, since it no longer shows the "final" version.